pkg/scheduler/scheduler.go

```diff
@@ -64,11 +65,13 @@ func (sched *Scheduler) Cache() internalcache.Cache {
 }
 
 type schedulerOptions struct {
-	schedulerName                  string
-	hardPodAffinitySymmetricWeight int32
-	disablePreemption              bool
-	percentageOfNodesToScore       int32
-	bindTimeoutSeconds             int64
+	schedulerName                    string
+	hardPodAffinitySymmetricWeight   int32
+	disablePreemption                bool
+	percentageOfNodesToScore         int32
+	bindTimeoutSeconds               int64
+	podInitialBackoffDurationSeconds int64
+	podMaxBackoffDurationSeconds     int64
```

Comment:
perhaps add the two new options to [TestSchedulerCreation](https://github.com/kubernetes/kubernetes/blob/a66200fe5de2a330d72298f05306f238d3a24ce8/pkg/scheduler/scheduler_test.go#L203)